src/controllers/_account_controllers.py

```diff
@@ -0,0 +1,31 @@
+from flask import request
+from src.config.soap_client import soap_client
+
+
+def register_handler():
+    try:
+        # Get JSON data from the request
+        data = request.json
+
+        if not data:
+            return {"msg": "No JSON data provided in the request"}, 400
+
+        username = data.get("username")
+        password = data.get("password")
+
+        if not username or not password:
+            return {"msg": "Required fields are missing in JSON data"}, 400
+
+        result = soap_client.service.account_register(
+            {"username": username, "password": password}
+        )
+
+        if result.auth is not None:
+            jwt = result.auth.token
+            return {"msg": "Register succeeded", "token": jwt}, 200
+
+        return {"msg": "Username already registered"}, 409
```

Comment:
Note that, if this block is reached, it is not guaranteed that the error was caused because the username is already in use. 

There is an open [issue](https://github.com/hawks-atlanta/authentication-go/issues/49) in the authentication repository about this, so you don't need to change it now. But keep this in mind in case you face some unexpected behavior.

Reply:
Thank you very much for the review, I will keep it in mind for when it is ready. Wow I did not realize that the percentage increased, thank you